src/helm.repoExplorer.ts

```diff
@@ -160,31 +160,19 @@ async function listHelmRepos(): Promise<Errorable<HelmRepo[]>> {
 }
 
 async function listHelmRepoCharts(repoName: string, repoUrl: string): Promise<Errorable<HelmRepoChart[]>> {
-    const indexUrl = vscode.Uri.parse(repoUrl).with({ path: 'index.yaml' }).toString();
-    try {
-        const charts = await listHelmRepoChartsCore(repoName, indexUrl);
-        return { succeeded: true, result: charts };
-    } catch (e) {
-        if (vscode.Uri.parse(repoUrl).authority.indexOf("127.0.0.1") >= 0) {
-            const d = await helm.helmServe();
-            try {
-                const charts = await listHelmRepoChartsCore(repoName, indexUrl);
-                return { succeeded: true, result: charts };
-            } catch (e2) {
-                return { succeeded: false, error: [ "" + e2 ] };
-            } finally {
-                d.dispose();
-            }
-        }
-        return { succeeded: false, error: [ "" + e ] };
+    const sr = await helm.helmExecAsync(`search ${repoName} -l`);
```

Comment:
I would recommend adding a `/` to the end of repoName. Otherwise, it is possible that the regexp matcher make return a greedier result set than you might expect.

For example, say I had a repo named `foo` and one named `foobar`. `helm search foo` would return results for both repos, while `helm search foo/` will only return those for repo `foo`.